### DISCUSSION TITLE
现在阿里云盘视频播放链接超时时间不是15min了吗

### DISCUSSION THREAD

**Discussion by Deepseaon** [AlistGo/alist#6215] [General]
最近看视频发现比如拉取链接到vlc这样的程序看的话之前是15分钟就卡住要重启续播，现在20多分钟的视频看到完也不卡了

**Comment by anwen-anyi** [comment]
嘿嘿 不是15分钟了不好么~ 
- https://github.com/alist-org/alist/pull/5560
